amd/src/eula_response.js

```diff
@@ -44,15 +44,15 @@ define(['jquery', 'core/str'], function($, str) {
                     data: {action: "accept_eula", sesskey: M.cfg.sesskey},
                     success: function() {
                         str.get_string('eulaaccepted', 'plagiarism_turnitinsim').done(function(text) {
-                            $('.eulacontainer').hide().html(text).fadeIn();
+                            $('.turnitinsim_eulacontainer').hide().html(text).fadeIn();
```

Comment:
I generally prefer finding element by id but I am fine here as it was searching with a class name before.